### DISCUSSION TITLE
Flip/transpose igx grid

### DISCUSSION THREAD

**Discussion by SkylerLutz** [IgniteUI/igniteui-angular#11177] [General]
## Question  
Hi -- I'm wondering if it is possible to make a grid to display its columns as rows and vice versa. I have a grid that can grow very wide (i.e. dozens of columns) and many of our users would like the ability to flip the grid so they can scroll vertically to their cell rather than having to scroll horizontally. Thanks!

I noticed "list view" is mentioned here but I can't find anything about it in the official ignite documentation: 
https://github.com/IgniteUI/igniteui-angular/wiki/igxGrid-Specification#end-user-experience

**Comment by ChronosSF** [comment]
Hi @SkylerLutz ,

Nothing out of the box but should be a straightforward function to transform the data so that rows become columns and vice versa. The grid is fully runtime rebindable so it will be as simple as passing new objects for columns and data .

Let me know if you have issues with this.
